cms/envs/common.py

```diff
@@ -1981,6 +1981,17 @@
     }
 ]
 
+LIBRARY_PROBLEM_TYPES = [
```

Comment:
This is actually not a problem type, rather it could be named `LIBRARY_BLOCK_TYPES` or `LIBRARY_XBLOCK_TYPES`

Reply:
updated